notebooks/Composer_Functional.ipynb

```diff
@@ -0,0 +1,306 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# install composer, hiding output to keep the notebook clean\n",
+    "# ! pip install mosaicml > /dev/null 2>&1\n",
+    "# To install from source instead of the last release, use this command instead:|\n",
+    "! pip install git+https://github.com/mosaicml/composer.git@dev > /dev/null 2>&1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using the Functional API\n",
+    "\n",
+    "In this tutorial, we'll see an example of using Composer's algorithms in a standalone fashion, with no changes to the surrounding code and no requirement to use the Composer trainer. We'll be training a simple model on CIFAR-10, similar to the [PyTorch classifier tutorial](https://pytorch.org/tutorials/beginner/blitz/cifar10_tutorial.html). Because we'll be using a toy model trained for only a few epochs, we won't get the same speed or accuracy gains we might expect from a more realistic problem. However, this notebook should still serve as a useful illustration of how to use various algorithms. For examples of more realistic results, see the MosaicML [Explorer](https://app.mosaicml.com/explorer/imagenet).\n",
+    "\n",
+    "First, we need to define our original model, dataloader, and training loop. Let's start with the dataloader:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "import torch.nn.functional as F\n",
+    "import torchvision\n",
+    "import torchvision.transforms as transforms\n",
+    "\n",
+    "datadir = './data'\n",
+    "batch_size = 1024\n",
+    "torch.manual_seed(42)\n",
+    "\n",
+    "\n",
+    "def make_dataloaders():\n",
```

Comment:
should we un-function-ify this (ironic because its the functional notebook 😂 ) so users can poke around in the variables interactively?

Reply:
done